Apps/Src/Voltage.c

```diff
@@ -50,7 +62,8 @@ ErrorStatus Voltage_Init(cell_asic *boards){
  */
 ErrorStatus Voltage_UpdateMeasurements(void){
 	int8_t error = 0;
-	
+	CPU_TS ts;
+	OSMutexPend(Voltage_Mutex, 0, OS_OPT_PEND_BLOCKING, &ts, &err);
```

Comment:
Can you move this to right before the function starts modifying `VoltageVal` (right before the `for` loop)? We don't want to take the mutex for longer than we need to.